sdk/anomalydetector/ai-anomaly-detector/package.json

```diff
@@ -70,7 +70,7 @@
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
-    "@azure/identity": "2.0.0-beta.6",
+    "@azure/identity": "^2.0.1",
```

Comment:
No `rush update`?

Reply:
Wouldnt the automation take care of that?

Reply:
If the version already exists in the pnpm-lock file, there is no need to do `rush update`.